Doc/library/http.rst

```diff
@@ -106,6 +106,7 @@ Code    Enum Name                           Details
 ``428`` ``PRECONDITION_REQUIRED``           Additional HTTP Status Codes :rfc:`6585`
 ``429`` ``TOO_MANY_REQUESTS``               Additional HTTP Status Codes :rfc:`6585`
 ``431`` ``REQUEST_HEADER_FIELDS_TOO_LARGE`` Additional HTTP Status Codes :rfc:`6585`
+``451`` ``UNAVAILABLE_FOR_LEGAL_REASONS``   Status Code to Report Legal Obstacles :rfc:`7725`
```

Comment:
Perhaps you should change it to the title of the rfc for consistency?
```suggestion
``451`` ``UNAVAILABLE_FOR_LEGAL_REASONS``   An HTTP Status Code to Report Legal Obstacles :rfc:`7725`
```

Reply:
Okay.